Resources/config/services.xml

```diff
@@ -5,35 +5,74 @@
+            <argument type="service" id="doctrine.migrations.dependency_factory"/>
+
+            <tag name="console.command"  command="doctrine:migrations:execute"/>
         </service>
-        <service id="doctrine_migrations.generate_command" class="Doctrine\Bundle\MigrationsBundle\Command\MigrationsGenerateDoctrineCommand">
-            <tag name="console.command" />
+        <service id="doctrine_migrations.generate_command" class="Doctrine\Migrations\Tools\Console\Command\GenerateCommand">
+            <argument>doctrine:migrations:generate</argument>
+            <argument type="service" id="doctrine.migrations.dependency_factory"/>
+
+            <tag name="console.command"  command="doctrine:migrations:generate"/>
         </service>
-        <service id="doctrine_migrations.latest_command" class="Doctrine\Bundle\MigrationsBundle\Command\MigrationsLatestDoctrineCommand">
-            <tag name="console.command" />
+        <service id="doctrine_migrations.latest_command" class="Doctrine\Migrations\Tools\Console\Command\LatestCommand">
+            <argument>doctrine:migrations:latest</argument>
+            <argument type="service" id="doctrine.migrations.dependency_factory"/>
+
+            <tag name="console.command"  command="doctrine:migrations:latest"/>
         </service>
-        <service id="doctrine_migrations.migrate_command" class="Doctrine\Bundle\MigrationsBundle\Command\MigrationsMigrateDoctrineCommand">
-            <tag name="console.command" />
+        <service id="doctrine_migrations.migrate_command" class="Doctrine\Migrations\Tools\Console\Command\MigrateCommand">
+            <argument>doctrine:migrations:migrate</argument>
+            <argument type="service" id="doctrine.migrations.dependency_factory"/>
+
+            <tag name="console.command"  command="doctrine:migrations:migrate" />
         </service>
-        <service id="doctrine_migrations.rollup_command" class="Doctrine\Bundle\MigrationsBundle\Command\MigrationsRollupDoctrineCommand">
-            <tag name="console.command" />
+        <service id="doctrine_migrations.rollup_command" class="Doctrine\Migrations\Tools\Console\Command\RollupCommand">
+            <argument>doctrine:migrations:rollup</argument>
+            <argument type="service" id="doctrine.migrations.dependency_factory"/>
+
+            <tag name="console.command" command="doctrine:migrations:rollup" />
         </service>
-        <service id="doctrine_migrations.status_command" class="Doctrine\Bundle\MigrationsBundle\Command\MigrationsStatusDoctrineCommand">
-            <tag name="console.command" />
+        <service id="doctrine_migrations.status_command" class="Doctrine\Migrations\Tools\Console\Command\StatusCommand">
+            <argument>doctrine:migrations:status</argument>
+            <argument type="service" id="doctrine.migrations.dependency_factory"/>
+
+            <tag name="console.command" command="doctrine:migrations:status" />
```

Comment:
Please add definition for commnad `doctrine_migrations.sync_metadata_command`.

I have created PR for you - https://github.com/goetas/DoctrineMigrationsBundle/pull/2/files